packages/frontend/src/features/errorBoundary/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,50 @@
+import { Flex, Text } from '@mantine/core';
+import { Prism } from '@mantine/prism';
+import * as Sentry from '@sentry/react';
+import { IconAlertCircle } from '@tabler/icons-react';
+import React, {
+    type ComponentProps,
+    type FC,
+    type PropsWithChildren,
+} from 'react';
+import SuboptimalState from '../../../components/common/SuboptimalState/SuboptimalState';
+
+export const ErrorBoundary: FC<
+    PropsWithChildren & { wrapper?: ComponentProps<typeof Flex> }
+> = ({ children, wrapper }) => {
```

Comment:
```suggestion
import { Flex, Text, type FlexProps } from '@mantine/core';
import { Prism } from '@mantine/prism';
import * as Sentry from '@sentry/react';
import { IconAlertCircle } from '@tabler/icons-react';
import { type FC, type PropsWithChildren } from 'react';
import SuboptimalState from '../../../components/common/SuboptimalState/SuboptimalState';

export const ErrorBoundary: FC<PropsWithChildren & { wrapper?: FlexProps }> = ({
    children,
    wrapper,
}) => {
```

you can just reference `FlexProps` instead of using `ComponentProps` but that's a just a NP of mine